cmd/routed-eni-cni-plugin/driver/driver.go

```diff
@@ -128,12 +132,53 @@ func (createVethContext *createVethPairContext) run(hostNS ns.NetNS) error {
 		return errors.Wrapf(err, "setup NS network: failed to set link %q up", createVethContext.contVethName)
 	}
 
-	// Add a connected route to a dummy next hop (169.254.1.1)
+	if createVethContext.v6Addr != nil && createVethContext.v6Addr.IP.To16() != nil {
+		//Enable v6 support on Container's veth interface.
+		if err = createVethContext.procSys.Set(fmt.Sprintf("net/ipv6/conf/%s/disable_ipv6", createVethContext.contVethName), "0"); err != nil {
+			if !os.IsNotExist(err) {
+				return errors.Wrapf(err, "setupVeth network: failed to enable IPv6 on container veth interface")
+			}
+		}
+
+		//Enable v6 support on Container's lo interface inside the Pod networking namespace.
+		if err = createVethContext.procSys.Set(fmt.Sprintf("net/ipv6/conf/lo/disable_ipv6"), "0"); err != nil {
+			if !os.IsNotExist(err) {
+				return errors.Wrapf(err, "setupVeth network: failed to enable IPv6 on container's lo interface")
+			}
+		}
+
+		//Enable v6 forwarding on Container's veth interface. We will set the same on Host side veth after we move it to host netns.
+		if err = createVethContext.procSys.Set(fmt.Sprintf("net/ipv6/conf/%s/forwarding", createVethContext.contVethName), "1"); err != nil {
```

Comment:
Agreed. Not needed for pod's internal intf. Since we're enabling it for the host side veth, I enabled it on the pod end of veth pair as well. Will remove it.